src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -285,6 +285,16 @@ public async Task InitializeAsync()
             _logger.LogTrace($"Done inferring Sql database schema in {timer.ElapsedMilliseconds}ms.");
         }
 
+        /// <inheritdoc />
+        public void InitializeAsync(
+            Dictionary<string, DatabaseObject> entityDatabaseObject,
+            Dictionary<string, string> graphQLStoredProcedureExposedNameToEntityNameMap)
+        {
+            GenerateDatabaseObjectForEntities();
```

Comment:
can you add comments for why this is necessary? arent the databse objects provided in the parameters via `entityDatabaseObject`